### ISSUE TITLE
BeeRef crashes on startup on Arch Linux

### ISSUE THREAD

**Issue by unfa** [rbreu/beeref#54]
**To Reproduce**
1. Start BeeRef
2. Have is crash

**Debug log:**

```
❯ beeref 
INFO beeref.__main__: Starting BeeRef version 0.2.0
INFO beeref.__main__: Using settings: /home/unfa/.config/BeeRef/BeeRef.ini
INFO beeref.__main__: Logging to: /home/unfa/.config/BeeRef/BeeRef.log
CRITICAL beeref.__main__: Unhandled exception
Traceback (most recent call last):
  File "/usr/bin/beeref", line 33, in <module>
    sys.exit(load_entry_point('BeeRef==0.2.0', 'gui_scripts', 'beeref')())
  File "/usr/lib/python3.10/site-packages/beeref/__main__.py", line 102, in main
    bee = BeeRefMainWindow(app)  # NOQA:F841
  File "/usr/lib/python3.10/site-packages/beeref/__main__.py", line 53, in __init__
    self.setWindowIcon(BeeAssets().logo)
  File "/usr/lib/python3.10/site-packages/beeref/assets/__init__.py", line 34, in __new__
    cls._instance.on_new()
  File "/usr/lib/python3.10/site-packages/beeref/assets/__init__.py", line 41, in on_new
    self.cursor_rotate = self.cursor_from_image(
  File "/usr/lib/python3.10/site-packages/beeref/assets/__init__.py", line 54, in cursor_from_image
    return QtGui.QCursor(pixmap, hotspot[0]/scaling, hotspot[1]/scaling)
TypeError: arguments did not match any overloaded call:
  QCursor(): too many arguments
  QCursor(QBitmap, QBitmap, hotX: int = -1, hotY: int = -1): argument 1 has unexpected type 'QPixmap'
  QCursor(QPixmap, hotX: int = -1, hotY: int = -1): argument 2 has unexpected type 'float'
  QCursor(Union[QCursor, Qt.CursorShape]): argument 1 has unexpected type 'QPixmap'
  QCursor(Any): too many arguments
Exception ignored in: <function BeeRefMainWindow.__del__ at 0x7f5fc1d7fd90>
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/beeref/__main__.py", line 60, in __del__
    del self.view
AttributeError: view
```

My system:

```
❯ inxi -F
System:    Host: unfa-desktop Kernel: 5.16.11-arch1-1 x86_64 bits: 64 Desktop: KDE Plasma 5.24.2 Distro: Arch Linux
Machine:   Type: Desktop Mobo: Gigabyte model: B550 AORUS ELITE V2 v: x.x serial: <superuser required>
           UEFI: American Megatrends LLC. v: F13 date: 07/08/2021
CPU:       Info: 12-Core model: AMD Ryzen 9 3900X bits: 64 type: MT MCP cache: L2: 6 MiB
           Speed: 3585 MHz min/max: 2200/3800 MHz Core speeds (MHz): 1: 3585 2: 3788 3: 3592 4: 3366 5: 3594 6: 3854 7: 3657
           8: 3320 9: 3860 10: 3599 11: 3584 12: 3593 13: 3596 14: 3586 15: 3586 16: 3970 17: 3896 18: 3592 19: 3593 20: 3595
           21: 3597 22: 3596 23: 3589 24: 3594
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] driver: amdgpu
           v: kernel
           Device-2: Logitech HD Pro Webcam C920 type: USB driver: snd-usb-audio,uvcvideo
           Device-3: MacroSilicon USB Video type: USB driver: hid-generic,snd-usb-audio,usbhid,uvcvideo
           Display: x11 server: X.Org 1.21.1.3 driver: loaded: amdgpu unloaded: modesetting,vesa resolution: 1: 1920x1080~60Hz
           2: 1920x1080~60Hz
           OpenGL: renderer: AMD Radeon RX 580 Series (POLARIS10 DRM 3.44.0 5.16.11-arch1-1 LLVM 13.0.1) v: 4.6 Mesa 21.3.7
Audio:     Device-1: AMD Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590] driver: snd_hda_intel
           Device-2: Advanced Micro Devices [AMD] Starship/Matisse HD Audio driver: snd_hda_intel
           Device-3: PreSonus Audio Studio 24c type: USB driver: snd-usb-audio
           Device-4: Logitech HD Pro Webcam C920 type: USB driver: snd-usb-audio,uvcvideo
           Device-5: MacroSilicon USB Video type: USB driver: hid-generic,snd-usb-audio,usbhid,uvcvideo
           Device-6: M-Audio Oxygen 49 type: USB driver: snd-usb-audio
           Sound Server-1: ALSA v: k5.16.11-arch1-1 running: yes
           Sound Server-2: PipeWire v: 0.3.47 running: yes
Network:   Device-1: Realtek RTL8125 2.5GbE driver: r8169
           IF: eno1 state: up speed: 1000 Mbps duplex: full mac: 18:c0:4d:ae:35:7d
           IF-ID-1: zt7nnmwu6v state: unknown speed: 10 Mbps duplex: full mac: 4e:06:00:9d:8c:52
Drives:    Local Storage: total: 5.69 TiB used: 3.91 TiB (68.8%)
           ID-1: /dev/sda vendor: Western Digital model: WD20EZRZ-00Z5HB0 size: 1.82 TiB
           ID-2: /dev/sdb vendor: Western Digital model: WD40EZRZ-22GXCB0 size: 3.64 TiB
           ID-3: /dev/sdc vendor: Transcend model: TS256GMTS400 size: 238.47 GiB
Partition: ID-1: / size: 1.82 TiB used: 572.1 GiB (30.7%) fs: btrfs dev: /dev/bcache0
           ID-2: /boot size: 475.9 MiB used: 93.2 MiB (19.6%) fs: ext4 dev: /dev/sdc4
           ID-3: /boot/efi size: 10 MiB used: 140 KiB (1.4%) fs: vfat dev: /dev/sdc1
           ID-4: /home size: 1.82 TiB used: 572.1 GiB (30.7%) fs: btrfs dev: /dev/bcache0
Swap:      ID-1: swap-1 type: partition size: 64 GiB used: 1.62 GiB (2.5%) dev: /dev/sdc2
Sensors:   System Temperatures: cpu: 32.0 C mobo: 30.0 C gpu: amdgpu temp: 32.0 C
           Fan Speeds (RPM): N/A gpu: amdgpu fan: 762
Info:      Processes: 822 Uptime: 1d 18h 47m Memory: 31.35 GiB used: 15.55 GiB (49.6%) Shell: fish inxi: 3.3.08
```

**Comment by ncmprhnsbl** [comment]
just adding a metoo (arch etc..)
and an unhelpful: probably to do with python 3.10 .. and similarly: https://github.com/rbreu/beeref/issues/49

**Comment by KannebTo** [comment]
I made PR #59 to fix this. It works for me on ArchLinux.

**Comment by AlansCodeLog** [comment]
Was also having this problem, tested PR, also on Arch, works perfectly now!

**Comment by hak0** [comment]
Same problem here on Mac OS. The PR works! Thanks!

**Comment by MegaKeegMan** [comment]
PR #59 fixed it for me on ArchLinux as well. Thanks very much!

**Comment by rbreu** [comment]
Forgot to close the issue. This has been fixed since 0.3.1.
